LeetCodeQuestions/LeetCodeQuestions/GoodPairs/GoodPairs.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+namespace LeetCodeQuestions
+{
+    public class GoodPairs
+    {
+
+        public static int NumIdenticalPairs(int[] nums)
+        {
+            int count = 0;
+            for (int i = 0; i < nums.Length; i++)
+            {
+                for (int j = 0; j < nums.Length; j++)
```

Comment:
you can solve this with O(n) without using a nested loop 
think about how you can optimize the solution without going to brute force

Reply:
similar to this 
```
public static int NumIdenticalPairs(int[] nums)
        {
            var dict = new Dictionary<int, int>();
            for (int i = 0; i < nums.Length; i++)
            {
                if (dict.ContainsKey(nums[i]))
                {
                    dict[nums[i]] += 1;
                }
                else
                {
                    dict[nums[i]] = 1;
                }
            }

            int k = 0;
            foreach (var keyValuePair in dict)
            {
                if (keyValuePair.Value == 1) continue;
                k += keyValuePair.Value * (keyValuePair.Value - 1) / 2;
            }

            return k;
        }
        ```

Reply:
Agree. Solved and committed to the new optimized solution